lib/src/dom/pretty_dom.dart

```diff
@@ -108,6 +113,6 @@ class PrettyDomOptions {
   /// You can configure this behavior by passing a custom `filterNode` function that should return true for every
   /// node that you wish to include in the output.
   /// {@endtemplate}
-  external bool Function(Node)? get filterNode;
-  external set filterNode(bool Function(Node)? value);
+  external bool Function(Node, dynamic, dynamic)? get filterNode;
+  external set filterNode(bool Function(Node, dynamic, dynamic)? value);
```

Comment:
If these extra args aren't expected, we could make them optional, so that if RTL JS ever doesn't pass them (either conditionally, or in future versions), it won't break this interop:
```suggestion
  external bool Function(Node, [dynamic, dynamic])? get filterNode;
  external set filterNode(bool Function(Node, [dynamic, dynamic])? value);
```

Reply:
ohhh that's a good idea!!!